standalone.tpl

```diff
@@ -0,0 +1,423 @@
+        name="sdg-preprocess",
+        image="{{exec_data_processing_op_image}}",
+        # This command will:
+        # 1. Fetch the SDG data
+        # 2. Preprocess the SDG data
+        # 3. Save the preprocessed data on the volume
+        command={{exec_data_processing_op_command}},
+        args={{exec_data_processing_op_args}},
+        # TODO: need another volume to read the model?
+        volume_mounts=get_sdg_vol_mount(),
+    )
+
+    volumes = [
+        kubernetes.client.V1Volume(
+            name=SDG_PVC_NAME,
+            persistent_volume_claim=kubernetes.client.V1PersistentVolumeClaimVolumeSource(
+                claim_name=SDG_PVC_NAME
+            ),
+        ),
+    ]
+
+    # TODO: append a command to copy the preprocessed data to the PVC
+
+    # Create and configure a spec section
+    template = kubernetes.client.V1PodTemplateSpec(
+        metadata=kubernetes.client.V1ObjectMeta(labels={"app": "sdg-preprocess"}),
+        spec=kubernetes.client.V1PodSpec(
+            restart_policy="Never",
+            init_containers=init_containers,
+            containers=[container],
+            volumes=volumes,
+        ),
+    )
+
+    # Create the specification of deployment
+    spec = kubernetes.client.V1JobSpec(
+        template=template,
+        ttl_seconds_after_finished=100,
+    )
+
+    # Instantiate the job object
+    job = kubernetes.client.V1Job(
+        api_version="batch/v1",
+        kind="Job",
+        metadata=kubernetes.client.V1ObjectMeta(name=job_name, namespace=namespace),
+        spec=spec,
+    )
+
+    return job
+
+
+def run_job(namespace: str, job: kubernetes.client.V1Job):
+    """
+    Create and run a Kubernetes job in the specified namespace, and wait for its completion.
+
+    Args:
+        namespace (str): The namespace in which to create the job.
+        job (kubernetes.client.V1Job): The job object to be created and run.
+
+    Prints:
+        str: The name of the created job.
+        str: The status of the job during its execution.
+        str: The logs of the pod if the job fails.
+
+    The function will print the job's status as it progresses and will stop watching once the job
+    either succeeds or fails. If the job fails, it will also print the logs of the failed pod.
+    """
+    # Create a job
+    batch_v1 = kubernetes.client.BatchV1Api()
+    resp = batch_v1.create_namespaced_job(body=job, namespace=namespace)
+    logger.info("Job created. status='%s'", resp.metadata.name)
+
+    # Wait for the job to complete
+    w = kubernetes.watch.Watch()
+    for event in w.stream(batch_v1.list_namespaced_job, namespace=namespace):
+        job = event["object"]
+        logger.info("Job: %s - %s", job.metadata.name, job.status.phase)
+        if job.status.phase == "Succeeded":
+            logger.info("Job completed successfully.")
+            w.stop()
+        elif job.status.phase == "Failed":
+            core_v1 = kubernetes.client.CoreV1Api()
```

Comment:
This keep throwing `AttributeError: 'V1JobStatus' object has no attribute 'phase'` Maybe the below is a solution. 
```suggestion
        if job.status.succeeded == 1:
            logger.info("Job completed successfully.")
            w.stop()
        elif job.status.failed == 1:
            core_v1 = kubernetes.client.CoreV1Api()
```

Reply:
Sure thanks